wapitiCore/attack/mod_wapp.py

```diff
@@ -60,6 +60,14 @@
 )
 
 
+def _is_valid_json(response):
+    try:
+        json.loads(response.content)
+        return True
```

Comment:
Pas nécessaire, car `not <json.loads()>` sera false. Tu pourrais donc return json.loads